liquidcore/home/templates/homepage.html

```diff
@@ -1,7 +1,8 @@
 {% extends "base.html" %}
 
 {% block content %}
-<h1>Liquid Investigations</h1>
+<h1 class="shady">Liquid Investigations</h1>
+<h1>{{ liquid_title }}</h1>
```

Comment:
Yeah that's super impolite. Use a single `<h1>` element within the page. The vertical one can be a `<p>`, or better yet, [`<aside>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside).

Reply:
this will get replaced with a decent theme anyway